### PR TITLE
Add tag plugin documentation for Wavedrom.

### PR DESCRIPTION
**Pull request by r12f**
This change adds documentation for next theme wavedrom tag plugin. It contains the syntax, configuration and example on how to use it.

### CODE REVIEW

**Comment by r12f** [comment]
workflow fails, but doesn't look like related to this change. Something in the test infra might need to be updated:

image

File: source/docs/tag-plugins/wavedrom.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+```jinja
+{% wavedrom %}
+{% endwavedrom %}
+```
+
+or
+
+~~~markdown
+```wavedrom
```
Comment (stevenjoezhang) [inline source/docs/tag-plugins/wavedrom.md]: It seems that wavedrom does not support this syntax

Comment (r12f) [reply source/docs/tag-plugins/wavedrom.md]: Ouch! You are right! I debugged a bit more. Although the hexo tag plugin doc mentioned the [backtick code block](https://hexo.io/docs/tag-plugins#Backtick-Code-Block). But the rendering doesn't really process them with tag extends. Only the swigs will be processed. Let me update the docs.

Comment (r12f) [reply source/docs/tag-plugins/wavedrom.md]: doc is updated now.

Comment (r12f) [reply source/docs/tag-plugins/wavedrom.md]: PR description is also updated.

**Comment by stevenjoezhang** [comment]
workflow fails because the hexo-theme-next used in this repository does not support waveform. We will release a new version next month and merge it.

**Comment by r12f** [comment]
woot! thanks!

**Pull request merged**